src/utils/copyExternalArtifacts.ts

```diff
@@ -0,0 +1,25 @@
+import path from 'path'
+import fs from 'fs'
+
+/**
+ * Copy external artifacts to the local artifacts folder
+ * This is a temporary hack until multiple artifacts paths are allowed
+ * @param externalArtifactsPath 'node_modules/@aragon/abis/os/artifacts'
+ */
+export function copyExternalArtifacts(
+  externalArtifactsPath: string,
+  localArtifactsPath = 'artifacts'
+): void {
+  const fromDir = path.resolve(externalArtifactsPath)
+  const toDir = path.resolve(localArtifactsPath)
+  if (!fs.existsSync(fromDir))
+    throw Error(
+      `Can not copy external artifacts from ${fromDir}, does not exist`
```

Comment:
Depending on if we catch and handle this error, I can see this being nice to just silently be ignored—if you don't have a local artifacts folder, it just goes on assuming you don't need it.

Reply:
How we are using it now it's critical that the folder exists, otherwise the plugin will break down the line with really obscure errors. I would leave the error for now and since this is a not nice solution it shouldn't last for long

Reply:
Sounds good to me!